content/p-vs-np/en/p-vs-np.md

```diff
@@ -56,9 +56,9 @@ A *witness* in computer science is proof that you solved the problem correctly.
 We will see later that a witness does not necessarily have to be the solution to the original problem. It could also be a solution for an alternative representation of the same problem.
 
 ### Problems in PSPACE: Not all problems have solutions that can be efficiently verified
-Problems that require exponential resources to solve and verify are called problems in PSPACE. The reason they are called PSPACE is because although they might take exponential time to solve, they don’t require exponential memory space to run the search.
+Problems that require exponential resources to solve and verify are called problems in PSPACE. The reason they are called PSPACE is that although they might take exponential time to solve, they don’t necessarily require exponential memory space to run the search.
 
-This class of problems has been researched extensively, yet no efficient algorithm to solve them has been discovered. Many researchers believe no efficient algorithm to solve these problems exists at all. If an efficient solution to these problems could be discovered, it would also be possible to reuse the algorithm to break all modern encryption and fundamentally alter computing as we know it.
+This class of problems has been researched extensively, yet no efficient algorithm to solve them has been discovered. Many researchers believe no efficient algorithm to solve these problems exists at all. If an efficient solution to these problems was discovered, it would also be possible to reuse the algorithm to break all modern encryption and fundamentally alter computing as we know it.
```

Comment:
For this line change, let's keep the original. To be precise, even if such a solution "could be" discovered, it doesn't matter if no-one discovers them.

Reply:
it sounds rather.. funny this way. but ok, will fix! Thanks